specifyweb/frontend/js_src/lib/components/TreeView/TreeDefaults.tsx

```diff
@@ -0,0 +1,524 @@
+  name: "Order",
+  title: "Order",
+  isEnforced: true,
+  isInFullName: false,
+  rankId: 100,
+}
+
+const suborder: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Suborder",
+  title: "Suborder",
+  isEnforced: false,
+  isInFullName: false,
+  rankId: 110,
+}
+
+const infraorder: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Infraorder",
+  title: "Infraorder",
+  isEnforced: false,
+  isInFullName: false,
+  rankId: 120,
+}
+
+const family: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Family",
+  title: "Family",
+  isEnforced: true,
+  isInFullName: false,
+  rankId: 140,
+}
+
+const subfamily: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Subfamily",
+  title: "Subfamily",
+  isEnforced: false,
+  isInFullName: false,
+  rankId: 150,
+}
+
+const tribe: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Tribe",
+  title: "Tribe",
+  isEnforced: false,
+  isInFullName: false,
+  rankId: 160,
+}
+
+const subtribe: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Subtribe",
+  title: "Subtribe",
+  isEnforced: false,
+  isInFullName: false,
+  rankId: 170,
+}
+
+const genus: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Genus",
+  title: "Genus",
+  isEnforced: true,
+  isInFullName: true,
+  rankId: 180,
+}
+
+const subgenus: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Subgenus",
+  title: "Subgenus",
+  isEnforced: false,
+  isInFullName: false,
+  rankId: 190,
+}
+
+const species: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Species",
+  title: "Species",
+  isEnforced: true,
+  isInFullName: true,
+  rankId: 220,
+}
+
+const subspecies: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Subspecies",
+  title: "Subspecies",
+  isEnforced: false,
+  isInFullName: true,
+  rankId: 230,
+}
+
+const variety: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Variety",
+  title: "Variety",
+  isEnforced: false,
+  isInFullName: true,
+  rankId: 240,
+}
+
+const subvariety: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Subvariety",
+  title: "Subvariety",
+  isEnforced: false,
+  isInFullName: true,
+  rankId: 250,
+}
+
+const forma: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Forma",
+  title: "Forma",
+  isEnforced: false,
+  isInFullName: true,
+  rankId: 260,
+}
+
+const subforma: Partial<SerializedResource<TaxonTreeDefItem>> = {
+  _tableName: "TaxonTreeDefItem",
+  name: "Subforma",
+  title: "Subforma",
```

Comment:
@grantfitzsimmons should default tree ranks be localized?

Reply:
I don't think so.